src/layouts/base-layout/index.tsx

```diff
@@ -6,7 +6,7 @@
 // Third-party imports
 import classNames from 'classnames'
 import AlertBanner from '@hashicorp/react-alert-banner'
-import { HTMLAttributes, useState } from 'react'
+import { HTMLAttributes, useEffect, useState } from 'react'
```

Comment:
I don't think the `useEffect` is used? But as long as the linter doesn't complain and it builds I have no issue with it.